examples/MvcWeb/TheraLang/Controllers/PaginationController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using MvcWeb.Db;
+using MvcWeb.TheraLang.Entities;
+using MvcWeb.TheraLang.Pagination;
+namespace MvcWeb.TheraLang.Controllers
+{
+    [Route("api/pag/projects")]
+    [ApiController]
+    public class PaginationController : Controller
+    {
+        IttmmDbContext context;
+        public PaginationController(IttmmDbContext Context)
+        {
+            this.context = Context;
+        }
+        [HttpGet]
+        public async Task<IActionResult> Index(int page = 1)
+        {
+            int pageSize = 5;
```

Comment:
hardcoded page size

Reply:
what exactly are u talking about? If we didn`t send number of page explicitly, it goes on page 1,
about page size? - i can change it

Reply:
> hardcoded page size

what exactly are u talking about? If we didn`t send number of page explicitly, it goes on page 1,
about page size? - i can change it

Reply:
he means that you should move it to Constans file, please add file Constants.cs into MVCWeb -> TheraLang folder and code inside
`public static class PagingConstants
    {
        public const int StartPage= 1;
        public const int ItemsPerPage = 10;
    }`